README.md

```diff
@@ -0,0 +1,302 @@
+# OncodriveCLUSTL
+
+OncodriveCLUSTL is a sequence-based clustering method to identify significant
+clustering signals in nucleotide sequence.
+
+One of the main goals of cancer research is the identification of the genomic
+elements that drive tumorigenesis. OncodriveCLUSTL is a new nucleotide
+sequence-based clustering algorithm to detect significant clustering signals
+across genomic regions. OncodriveCLUSTL is based on a local background model
+derived from the nucleotide context mutational probabilities of the cohort under
+study. Our method is able to identify well-known cancer drivers in coding
+regions and it can be applied to non-coding regions and non-human data.
+
+## License
+
+OncodriveCLUSTL is available to the general public subject to certain conditions
+described in its [LICENSE](LICENSE).
+
+
+## Installation
+
+OncodriveCLUSTL depends on Python 3.5 or above. We recommend to install it using
+the [Anaconda Python distribution](https://www.anaconda.com/download/):
+
+```bash
+conda install -c bbglab oncodriveclustl
+```
+
+OncodriveCLUSTL can also be installed using pip:
+
+```bash
+pip install oncodriveclustl
+```
+
+You can obtain the latest code from the repository and install it for development with pip:
+
+```bash
+git clone git@bitbucket.org:bbglab/oncodriveclustl.git
+cd oncodriveclustl
+python -m venv .venv
+.venv/bin/pip install -e .
+source .venv/bin/activate
+oncodriveclustl --help
+```
+
+> [!NOTE]
+> The first time that you run OncodriveCLUSTL with a given reference genome, it
+> will download it from our servers. By default the downloaded datasets go to
+> `~/.bgdata`. If you want to move these datasets to another folder you have to
+> define the system environment variable `BGDATA_LOCAL` with an export command.
+
+> [!NOTE]
+> If you install a modern build tool like [rye](https://rye.astral.sh), you can
+> simply do this:
+> ```bash
+> git clone git@bitbucket.org:bbglab/oncodriveclustl.git
+> cd oncodriveclustl
+> rye sync
+> rye run oncodriveclustl --help
+> ```
+
```

Comment:
should this be replaced with uv? or are the two interchangable?

Reply:
Thanks! I missed that. Fixed.